providers/core/core.oauth.js

```diff
@@ -98,8 +100,11 @@ OAuth.prototype.launchAuthFlow = function(authUrl, stateObj, continuation) {
     });
     return;
   }
-
-  this.ongoing[stateObj.state].launchAuthFlow(authUrl, stateObj, continuation);
+  if (interactive === undefined) {
+    interactive = true;
```

Comment:
Yup - defaulting interactive=true will give the current behavior that freedom-social-xmpp and the Google+ provider in freedom-social-firebase rely on.